.github/workflows/ci-pull-request-cspm-org.yaml

```diff
@@ -0,0 +1,48 @@
+name: CI - Pull Request CSPM Org
```

Comment:
non blocker, but these 4 `ci-pr-request-*` look like could be one single file, with several path.
you could detect the path and use it when required instead of needing to maintain 4 ci files.

Reply:
is it possible to include multiple paths and run a step or other depending on the modified path? I thought that path was for the whole workflow not for each step/job

Reply:
I added another step for building org templates in existing file as both single and org templates have same path. let me know if this looks good. cc @wideawakening / @hayk99